### PR TITLE
Ars Magica - Combat Rollers

### PR DESCRIPTION
**Pull request by Medieve**
Brackets and parenthesis were out of order, messing with how the modifiers were being added (or in this, not at all)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -235,3 +235,4 @@ Call_of_Cthulhu_Spanish/Delta Green Variant
 /base.css
 /app.css
 /.vs/config/applicationhost.config
+WFRP4CharSheet/WFRP4Charsheet_beforebutcher.html
```
Comment (Anduh) [inline .gitignore]: I doubt this edit was mean to be included

Comment (Medieve) [reply .gitignore]: Oops, no its not. Uh, not sure how to remove the file from the submission.

Comment (Anduh) [reply .gitignore]:
```suggestion
```
maybe this works

Comment (Anduh) [reply .gitignore]: If you approve the above code change the problem should be solved

**Pull request merged**